release 3.0.0 (#1250)
microsoft/onefuzz: CHANGELOG.md

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## 3.0.0
### Changed

> * CLI/Deployment/Service: Move to using `api://` for AAD Application "identifier uris". [#1243](https://github.com/microsoft/onefuzz/pull/1243)

Reviewer: ```suggestion
* CLI/Deployment/Service: Move to using `api://` for AAD Application "identifier URIs". [#1243](https://github.com/microsoft/onefuzz/pull/1243)
```

> * Agent/Supervisor/Proxy: Replace machine name in runtime statistics reported to Microsoft via Application Insights with with "Redacted".  [#1242](https://github.com/microsoft/onefuzz/pull/1242)

Reviewer: ```suggestion
* Agent/Supervisor/Proxy: Redact device, IP, and machine name in runtime statistics reported to Microsoft via Application Insights. [#1242](https://github.com/microsoft/onefuzz/pull/1242)
```
